tests/functional/sse-kms-migration/helpers.js

```diff
@@ -14,20 +37,121 @@ function getKey(key) {
     return config.kmsHideScalityArn ? getKeyIdFromArn(key) : key;
 }
 
-// for Integration use default profile, in cloudserver use vault profile
+// For Integration use default profile, in cloudserver use vault profile
 const credsProfile = process.env.S3_END_TO_END === 'true' ? 'default' : 'vault';
-const s3config = getConfig(credsProfile, { signatureVersion: 'v4' });
-const s3 = new S3(s3config);
+
+// Create custom agents with specific pooling settings
```

Comment:
```suggestion
```

if you want to make it clear, you can change the name of the var `httpAgent` with `httpAgentWithSpecificPooling` if this is important when we read tests